applications/luci-app-keepalived/Makefile

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2022 Jaymin Patel <jem.patel@gmail.com>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI support for the Keepalived
+LUCI_DEPENDS:=+luci-base +keepalived +keepalived-sync
```

Comment:
Do we really need `keepalvied-sync` as a dependency?

Reply:
Yes, now it also has an option to configure HA peers synchronization options.

https://github.com/openwrt/luci/pull/5965#issuecomment-1249823590